Source/UI/NewUI/BillingForm/Style/Default Implementation/BillingForm/Header/Cancel/DefaultCancelButtonFormStyle.swift

```diff
@@ -0,0 +1,36 @@
+import UIKit
+
+struct DefaultCancelButtonFormStyle: FormButtonStyle {
+    var isEnabled: Bool
+    var text: String
+    var font: UIFont
+    var activeTitleColor: UIColor
+    var inActiveTitleColor: UIColor
+    var inActiveTintColor: UIColor
+    var activeTintColor: UIColor
+    var backgroundColor: UIColor
+    var height: Double
+    var width: Double
+    
+    init( isEnabled: Bool = true,
+          text: String = "Cancel",
+          font: UIFont = UIFont.systemFont(ofSize: 17),
+          activeTitleColor: UIColor = .brandeisBlue,
+          inActiveTitleColor: UIColor = .doveGray,
+          inActiveTintColor: UIColor = .doveGray,
+          activeTintColor: UIColor = .brandeisBlue,
+          backgroundColor: UIColor = .white,
+          height: Double = 44,
+          width: Double = 53) {
```

Comment:
Good catch @alex-ioja-yang-cko . We did discuss this over our last refinement. We are waiting for our designer feedback. I think we do need some min height & width for all fields which will be part of a separate ticket and also allow merchants to pass in their own width and height.. Something which we did not discuss this on ACs for this ticket. 

@ehab-al-cko @harry-brown-cko you can keep that in mind so that you are aware of this change coming along your way.

Reply:
Yes, it's safe